includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -18238,6 +18306,7 @@ class AMP_Allowed_Tags_Generated {
 	private static $layout_allowed_attrs = array(
 		'data-amp-bind-height' => array(),
 		'data-amp-bind-width' => array(),
+		'disable-inline-width' => array(),
```

Comment:
This is new. Docs: https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/#disable-inline-width

I wonder, does this mean that the `sizes` attribute can cease to be removed in favor of setting this boolean attribute?

https://github.com/ampproject/amp-wp/blob/014bbbba04bea102387f6faaa6b56ab8bf0e1e0a/includes/sanitizers/class-amp-img-sanitizer.php#L333-L334

Reply:
Asked here: https://github.com/ampproject/amphtml/pull/27083#issuecomment-610767867

Reply:
Answer is yes! https://github.com/ampproject/amphtml/pull/27083#issuecomment-610961584

So in a subsequent PR we should eliminate the `unset()` here and instead add the `disable-inline-width` attribute. This would be done on the `develop` branch to give it some time to get tested.

Reply:
Is this being tracked in a new issue?